plugins/search-backend-module-pg/api-report.md

```diff
@@ -43,7 +44,7 @@ export class DatabaseDocumentStore implements DatabaseStore {
   // (undocumented)
   query(
     tx: Knex.Transaction,
-    { types, pgTerm, fields, offset, limit }: PgSearchQuery,
+    { types, pgTerm, fields, offset, limit, options }: PgSearchQuery,
```

Comment:
these should probably have been just `query: PgSearchQuery` in the implementation so this diff line wouldn't have had to exist